src/vueDragulaConfig.ts

```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue';
 import 'dragula/dist/dragula.css';
 
+// @ts-ignore
 import { Vue2Dragula } from 'vue2-dragula';
```

Comment:
This dependency does not have typescript type definition 😢

Reply:
That is a big oof